src/OpenColorIO/CPUProcessor.cpp

```diff
@@ -1680,7 +1680,7 @@ OCIO::ConstCPUProcessorRcPtr BuildCPUProcessor(OCIO::TransformDirection dir)
     OCIO::ConfigRcPtr config = OCIO::Config::Create();
 
     OCIO::MatrixTransformRcPtr transform = OCIO::MatrixTransform::Create();
-    const float offset4[4] = { 1.4002f, 0.4005f, 0.8007f, 0.5007f };
+    const double offset4[4] = { 1.4002, 0.4005, 0.8007, 0.5007 };
```

Comment:
should this be constexpr? I think there are a number of those in this PR.

Reply:
changed